### ISSUE TITLE
Need to install TauDem to use with QGIS

### ISSUE THREAD

**Issue by Robotuner** [dtarb/TauDEM#126]
Got a new machine, installed QGIS and found that TauDEM would fail with mpiexec is not recognized as an internal or external command.  After researching, I realize that TauDEM is a separate install (I guess I forgot).  Anyway, the windows installer at: http://hydrology.usu.edu/taudem/taudem5/downloads2.html does not download.  I think GitHub-cloud.s3.amazonaws.com isn't giving enough bandwidth to TauDEM.  It estimates > 12 hours and eventually fails.

Going to the plan 2, I am able to download TauDEM 5.3.7 Command Line Exec. 

- I can find GDAL 2.1.0 for python 2.7 from (https://pypi.python.org/pypi/GDAL/2.1.0). 

- I am unable to locate release-1800-x64-gdal-2-1-0-mapserver-7-0-1/gdal-201-1800-x64-core.msi, I can get gdal-111-1800-x64-core.msi, BUT I'M NOT SURE IF IT WILL work with python 2.7,  **Is it OK to install python 3.x instead**? 

-  I have the other dependencies. 

  I am unable to locate GDAL 201 for 64 bit windows from GISInternals.com, I can get gdal-111-1800-x64-core.msi, BUT I'M NOT SURE IF IT WILL work with python 2.7,  Is it OK to install python 3.x instead?  I have the other dependencies. 

Basically, I think all I need is the GDAL 201 unless I can substitute a newer version (which one?)

It would be easier if there were a way to get the windows installer.  If I can get it, I will be happy to post on my server for others to get.

**Comment by dtarb** [comment]
Try this link https://www.dropbox.com/s/zqr0upbcotj2vgd/TauDEM537_setup.exe as an alternative.

**Comment by Robotuner** [comment]
David:
 
Thanks, it was a great help.  I'm placing it on my server too.
 
[ http://www.engenious.com/Downloads/TauDEM/TauDEM537_Setup.exe ]( http://www.engenious.com/Downloads/TauDEM/TauDEM537_Setup.exe )
 
I also want to take this opportunity to thank you for the package.  It enables me to use QGIS as a front end to my hydrology package.  The benefit is that it gets me out of the graphical layout business and allows me to focus on hydrology.
 
Lenny
 
 
-----Original Message-----
